sdk/communication/AzureCommunicationCalling/Swift_Generated_Interfaces/ACSRenderer.h

```diff
@@ -0,0 +1,33 @@
+//+-----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Module name : ACSRenderer.h
+//
+//------------------------------------------------------------------------------
+
+public protocol RendererDelegate : NSObjectProtocol {
+
+    func rendererFailedToStart()
+
+    
+    optional func onFirstFrameRendered()
+}
+
+open class Renderer : NSObject {
+
+    
+    public init(localVideoStream: LocalVideoStream) throws
+
+    public init(remoteVideoStream: RemoteVideoStream) throws
+
+    open func createView() throws -> RendererView
+
+    open func createView(with options: RenderingOptions?) throws -> RendererView
+
+    open func dispose()
+
+    
+    open var size: StreamSize { get }
+
+    unowned(unsafe) open var delegate: RendererDelegate?
+}
```

Comment:
This implementation is tied to a particular UI paradigm. What are you targeting? Programmatic UIs, Storyboards, XIB, SwiftUI? Have you explored how well this works with these different approaches?

Reply:
We haven't tested ourselves with Storyboards, we have been using SwiftUI as of now. But story board has been tested by our customers and there were no issues they reported.

Reply:
That's great. SwiftUI was what I was most concerned about